integration/nwo/fabric/fpc/fpc.go

```diff
@@ -117,9 +117,9 @@ func (n *Extension) PostRun(load bool) {
 		return
 	}
 
-	if load {
-		// run docker containers
-	}
+	//if load {
```

Comment:
we can remove these lines completely, the docker containers are spawned in line 130 `n.deployChaincode(chaincode)`

Reply:
Sure, then the `load` variable is not used at all in this method. (I can't remove it from the signature though, because it's part of the interface it implements.)